wait-for-couchbase.sh

```diff
@@ -76,7 +76,7 @@ function createHotelsIndex() {
 }
 
 ##### Script starts here #####
-ATTEMPTS=150
+ATTEMPTS=15
```

Comment:
I assume the failure is evident after 15 tries, 150 seems excessive and made it harder to troubleshoot issues.

Reply:
yea, from my experience it was evident after 15